### DISCUSSION TITLE
How can I install code server  in Windows

### DISCUSSION THREAD

**Discussion by kumarsunilbesu** [coder/code-server#6868] [Q&A]
I am using 64-bit operating system, x64-based processor Windows 10. I am struggling to install code server through npm.
I tried Docker Desktop and used the image there. it is not able to run the jar bundled inside my vsix extension.
npm package worked for me on mac. similar solution i want do in my windows as well (do not want to go with docker and all)

**Comment by code-asher** [comment]
We have no support for running code-server on Windows at the moment but there is this issue: https://github.com/coder/code-server/issues/1397
